pkg/controller/operators/catalog/operator.go

```diff
@@ -1180,6 +1183,64 @@ func (o *Operator) unpackBundles(plan *v1alpha1.InstallPlan) (bool, *v1alpha1.In
 	return unpacked, out, nil
 }
 
+func (o *Operator) gcInstallPlans(log logrus.FieldLogger, namespace string) {
```

Comment:
What does `gc` stand for?

Reply:
garbage collect

Reply:
Could we document that somehow in the code?